Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:21.04 AS build
+
+RUN dpkg --add-architecture i386 && \
+  apt-get update && \
+  apt-get install -y curl
```

Comment:
I'm using a multi-stage build as we don't need curl itself in the actual runtime image.

Reply:
`curl` libraries are needed by Red.

Reply:
So we could instead `curl` instead of `libcurl4`?

Reply:
Ah, sure. It's even like that in [official docs](https://www.red-lang.org/p/download.html)

Reply:
👍